src/main/java/com/pusher/client/example/PrivateChannelExampleApp.java

```diff
@@ -66,10 +67,9 @@ public void onError(final String message, final String code, final Exception e)
     /* PrivateChannelEventListener implementation */
 
     @Override
-    public void onEvent(final String channelName, final String eventName, final String data) {
+    public void onEvent(final PusherEvent event) {
 
-        System.out.println(String.format("Received event [%s] on channel [%s] with data [%s]", eventName, channelName,
-                data));
+        System.out.println(String.format("Received event [%s] on channel [%s] with data [%s] and event [%s]", event.toString()));
```

Comment:
Missing strings to fill in the first three placeholders:
```suggestion
        System.out.println(String.format("Received event [%s] on channel [%s] with data [%s] and event [%s]", event.getEventName(), event.getChannelName(), event.getData(), event.toString()));
```